sentiment_analysis_tutorial.Rmd

```diff
@@ -0,0 +1,261 @@
+---
+title: "Sentiment Analysis Processing and Visualization Tutorial"
+author: "Lylybell Teran"
+output:
+  pdf_document:
+  html_document:
+    df_print: paged
+---
+
+```{r setup, include=FALSE}
```

Comment:
please remove the yaml header.

![1668461375494](https://user-images.githubusercontent.com/114460912/201769497-172d4754-ae23-4214-912d-e4823a0037f1.png)

Reply:
resolved